content/en/blog/_posts/2024-04-28-contribute-as-sig-docs-reviewer.md

```diff
@@ -0,0 +1,109 @@
+   Ricardo Amaro (Acquia)
+---
+
+You've probably come across the saying, "Contributing doesn't always mean writing lines of code"
+whispered to the wind or glimpsed in the margins of a sacred open-source tome.
+What is the truth behind this tale as old as time (or at least as old as Kubernetes itself)?
+
+Imagine Sarah: an enthusiast of open-source whose keyboard had seen more action crafting
+technical masterpieces for her projects than a knight's sword in battle. Sarah's had such
+a smooth relationship with Git, that she could `git commit` and `git push` in her sleep,
+and her etiquette in pull-requests was so polished, it could make a silver spoon look tarnished.
+
+But here's the twist: when it came to the mysterious _art-of-coding_, Sarah felt more
+like a fish out of water — or more accurately, like trying to navigate a Kubernetes cluster
+with a compass and a star map. The very thought of diving into the codebase made her feel like
+she was trying to read ancient hieroglyphs without a Rosetta Stone.
+
+Did Sarah let this stop her? Not at all. She discovered a path as impactful as a well-placed
+line of code: joining the Special Interest Group (SIG) Docs team. It was there that Sarah found her true calling,
+helping transform the dense field of Kubernetes documentation into a well-tended garden of knowledge.
+
+Sarah's odyssey from a mere mortal Kubernetes user to a demigod of documentation didn't
+just prove that contributions of knowledge and expertise are as valuable as code
+contributions—it also showed that you don't need to be a coding wizard to make a significant
+impact. Her journey is a beacon of hope for all who fear the merge conflict, a testament
+to the power of the written word in the realm of open-source.
+
+So, let Sarah's story inspire you. No matter if you're a code ninja or a prose maestro,
+there's a place for you in the open-source pantheon. And who knows? Maybe one day,
+your documentation will be the guiding light for a lost soul in the Kubernetes sea,
+making you the unsung hero of their open-source odyssey.
+
+## Why Become a SIG Docs Reviewer?
+
+The Kubernetes project depends substantially on the SIG Docs team to ensure that documentation
+is accurate, up to date, and easily accessible. By becoming a reviewer, you can help users
+and contributors navigate and understand Kubernetes more effectively. In addition, reviewing
+documentation provides unique opportunities for:
+
+- **Expand your Kubernetes knowledge**: Engage deeply with new features and functionalities
+  by reviewing their documentation.
+- **Improve your technical writing skills**: Develop an eye for detail and clarity in technical
+  writing.
+- **Strengthen the Kubernetes community**: Help maintain the high quality of Kubernetes
+  documentation, supporting both new and experienced users.
+- **Build your network**: Expanding your professional network, and getting together with
+  contributors from around the globe.
+
+## Who Are We Looking For?
```

Comment:
we try to avoid using "we" in writing, as per our style guide – lets replace this with "Who is SIG Docs looking for?"

Reply:
fixed upstream also